spec/features/categories_show_spec.rb

```diff
@@ -6,7 +6,7 @@
     sign_in @user
     @category = Category.create(name: 'Travel', icon: 'icon', user: @user)
     @transaction = Transaction.create(name: 'trip', amount: 344, user: @user)
-    visit '/categories/1'
+    visit '/categories/category_id'
   end
 
```

Comment:
- [ ] @H-Ramalan, great effort working on your project, but currently the test cases have some of it still failing and I would advise you to kindly work on it and be sure to have all the test cases passing, Below is a video demonstration of where the problem is coming from.

Video 👇 


https://github.com/H-Ramalan/dime-pay/assets/78634841/efac42bc-eb55-48bc-991d-2609a22a4a0e